enhancements/baremetal/baremetal-operator-management.md

```diff
@@ -0,0 +1,166 @@
+---
+title: baremetal-operator-management
+authors:
+  - "@markmc"
+reviewers:
+  - "@abhinavdahiya"
+  - "@dhellmann"
+  - "@enxebre"
+  - "@eparis"
+  - "@hardys"
+  - "@sadasu"
+  - "@smarterclayton"
+  - "@stbenjam"
+approvers:
+  - TBD
+creation-date: 2020-02-13
+last-updated: 2020-02-13
+status: provisional
+see-also:
+  - https://github.com/openshift/enhancements/pull/200
+replaces:
+superseded-by:
+---
+
+# Managing the Bare Metal Operator
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+1. Should the Bare Metal Operator be managed by the Cluster Version
+   Operator or the Machine API Operator?
+2. How to handle upgrades
+3. Which release this change should target
+
+## Summary
+
+The Bare Metal Operator provides bare metal machine management
+capabilities needed for the Machine API provider on the `baremetal`
+platform, and there is no equivalent for this component on other
+platforms.
+
+The Bare Metal Operator is currently managed by the Machine API
+Operator in a fashion that requires the Machine API Operator to have
+significant bare metal specific knowledge. This can be resolved either
+by:
+
+1. The Machine API Operator can add a generic operator management
+   framework (similar to some Cluster Version Operator capabilities)
+   for platform specific operators, and the Bare Metal Operator would
+   integrate with this framework.
+2. The Cluster Version Operator can gain the ability to manage
+   operators that are platform-specific, and the Bare Metal Operator
+   would move to being managed by the Cluster Version Operator.
+
+## Motivation
+
+In order to bring up a cluster using the baremetal platform:
+
+1. The installer needs to capture bare metal host information and
+   provisioning configuration for later use by baremetal-operator
+2. Something needs to install the CRDs for these bare metal specific
```

Comment:
These feel like they are part of MAO, just like CNO installs CRDs for the two core platform types.  In general, CNO already demonstrates this pattern and is successful doing so, so the default answer for this pattern is `MAO should behave like CNO` and any deviation needs justification.

Reply:
As above, I'll re-work this to propose bare metal having its own SLO.